cmd/nodecmd/create.go

```diff
@@ -199,6 +211,25 @@ func preCreateChecks(clusterName string) error {
 	if err := failForExternal(clusterName); err != nil {
 		return err
 	}
+	// bootsrap checks
+	if useEtnaDevnet && (len(bootstrapIDs) != 0 || len(bootstrapIPs) != 0 || genesisPath != "" || upgradePath != "") {
+		return fmt.Errorf("etna devnet uses predefined bootsrap configuration")
+	}
+	if len((bootstrapIDs)) != len(bootstrapIPs) {
+		return fmt.Errorf("number of bootstrap ids and ip:port pairs must be equal")
+	}
+	if genesisPath != "" && !utils.FileExists(genesisPath) {
+		return fmt.Errorf("genesis file %s does not exist", genesisPath)
+	}
+	if upgradePath != "" && !utils.FileExists(upgradePath) {
+		return fmt.Errorf("upgrade file %s does not exist", upgradePath)
+	}
+	// check ip:port pairs
+	for _, ip := range bootstrapIPs {
+		if _, err := netip.ParseAddrPort(ip); err != nil {
+			return fmt.Errorf("invalid ip:port pair %s", ip)
+		}
+	}
```

Comment:
Can we unit test these cases?

Also should we allow bootstrap IDs/IPs in the case Fuji/Mainnet?

Reply:
it's allowed. these params are available for user to use.
I feel like I should move it into `pkg/utils` so I can add unittest for verification `func`